exercises/accumulate/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "accumulate",
+  "name": "exercism-reasonml",
```

Comment:
This makes us change all package names to a common name but we can live with that. 😄